src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -719,16 +718,20 @@ public void testMitochondrialRefConf() throws Exception {
                 "-ERC", "GVCF",
                 "-LODB", "-2.0",
                 "-LODB", "0.0",
-                "-min-AF", "0.00");
+                "-min-AF", "0.00","-bamout", "/Users/davidben/Desktop/bamout-0.00.bam");
         runCommandLine(args3);
         final Pair<VCFHeader, List<VariantContext>> result_noThreshold = VariantContextTestUtils.readEntireVCFIntoMemory(unthresholded.getAbsolutePath());
 
         final Map<String, VariantContext> variantMap2 = result_noThreshold.getRight().stream().collect(Collectors.toMap(vc -> keyForVariant(vc), Function.identity()));
 
         //TLODs for variants should change too much for variant allele, should change significantly for non-ref
```

Comment:
Should there be a `not` in the first part of this comment?

TLODs for variants should **not** change too much for variant allele, should change significantly for non-ref

Reply:
Yes there should!

Reply:
fixed.